mesa_frames/abstract/space.py

```
@@ -436,29 +436,35 @@
                 elif isinstance(a, AgentsDF):
-                    ids.append(self._srs_constructor(a._ids, name="agent_id"))
+                    ids.append(
+                        self._srs_constructor(a._ids, name="agent_id", dtype="uint64")
+                    )
             return self._df_concat(ids, ignore_index=True)
         elif isinstance(agents, int):
-            return self._srs_constructor([agents], name="agent_id")
+            return self._srs_constructor([agents], name="agent_id", dtype="uint64")
         else:  # IDsLike
-            return self._srs_constructor(agents, name="agent_id")
+            return self._srs_constructor(agents, name="agent_id", dtype="uint64")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)